app/services/users.js

```diff
@@ -40,8 +55,19 @@ exports.findAllUsers = (limit, offset) =>
   User.findAndCountAll({
     limit,
     offset,
-    attributes: ['id', 'firstName', 'lastName', 'email']
+    attributes: ['id', 'firstName', 'lastName', 'email', 'role']
   }).catch(() => {
     logger.info('Error trying to find the users');
     throw errors.databaseError('Error processing request in database.');
   });
+
+exports.changeRole = email =>
+  User.update(
+    { role: ADMIN_ROLE },
```

Comment:
did you see my comment of making this method available to change to any role?
here you are making the change to admin role always, its not generic